documentation/Using-the-SDK/Advanced/Disposable-Extensions.md

```diff
@@ -1,37 +1,41 @@
 # Abstract
 
-This document outlines making your Custom Data Sources and Extensions
-disposable.
+This document outlines how to make a `CustomDataSource` or SDK
+extension - such as a `DataCooker` disposable.
```

Comment:
there should be one more dash after `DataCooker`

extension - such as a `DataCooker`  - disposable.

Reply:
thanks, don't know why I missed that